de_workloads/ingest/Ingest_AzureSql_Example/data_factory/adf_linked_services.tf

```diff
@@ -0,0 +1,22 @@
+resource "azurerm_data_factory_linked_custom_service" "ls_AzureSql_ExampleSource" {
+  name            = "ls_AzureSql_ExampleSource"
```

Comment:
name = "ls_AzureSql_ExampleSource". better to change this to variable

Reply:
done